cpp/log/frontend_test.cc

```diff
@@ -162,8 +163,11 @@ TYPED_TEST(FrontendTest, TestSubmitValid) {
   CertChain chain(this->leaf_pem_);
   EXPECT_TRUE(chain.IsLoaded());
 
+  LogEntry entry;
   SignedCertificateTimestamp sct;
-  EXPECT_OK(this->frontend_.QueueX509Entry(&chain, &sct));
+  EXPECT_OK(this->frontend_.QueueProcessedEntry(
+      this->submission_handler_.ProcessX509Submission(&chain, &entry), entry,
```

Comment:
Yep, that's a good idea and actually fits very well with what `QueueProcessedEntry` really does, I just kept things the same 'shape' as they were previously, but we can do this change in a separate PR.